.github/workflows/ci-synthetic-test.yml

```
@@ -0,0 +1,93 @@
+      - id: check
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const body = context.payload.pull_request?.body;
+            const syntheticTestMatch = body?.match(/\[(?<checked>[ x])\]\s+\*\*Spin up synthetic test environment\*\* <!-- #synthetic-test -->/);
+            const targetUrlMatch = body?.match(/Target URL:\s*(https?:\/\/[^\s\n]+)/i);
+            
+            // Check if this is a workflow_dispatch event
+            const isWorkflowDispatch = context.eventName === 'workflow_dispatch';
+            const dispatchTargetUrl = context.payload.inputs?.target_url;
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)